app.js

```diff
@@ -2,7 +2,8 @@ require('dotenv').config()
 
 var createError = require('http-errors')
 var express = require('express')
-var path = require('path')
+var cors = require('cors')
+// var path = require('path')
```

Comment:
we don't use 'path' anymore?

Reply:
not in this file, we use it in `routes/index.js`